client_cli/pytests/src/client_cli/iroha.py

```diff
@@ -21,6 +21,11 @@ def __init__(self, config: Config):
         :type path: str
         """
         super().__init__(config)
+        self._storage: Union[Dict, List] = {}
+        self._domains: Union[Dict, List] = {}
+        self._accounts: Union[Dict, List] = {}
+        self._assets: Union[Dict, List] = {}
+        self._asset_definitions: Dict[str, Any] = {}
```

Comment:
I removed this shared variables on purpose. 
Because we can use local variable.